### PR TITLE
Feature/more performant number grammar

### PR DESCRIPTION
**Pull request by sfgeorge**
A duplicate of adhearsion/ruby_speech#39

### CODE REVIEW

File: spec/support/dtmf_helper.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+#
+# Convert a simple DTMF string from "1 star 2" to "dtmf-1 dtmf-star dtmf-2".
+#
+# @param [Array] sequence A set of DTMF keys, such as `%w(1 star 2)`.
+#
+# @return [String] A string with "dtmf-" prefixed for each DTMF element.
+#                  Example: "dtmf-1 dtmf-star dtmf-2".
+#
+def dtmf_seq(sequence)
+  sequence.map { |d| "dtmf-#{d}" }.join " "
```
[the remark this answers is not in the document]
Comment (sfgeorge) [reply spec/support/dtmf_helper.rb]: luz @ hound.  You changed your policy recently, didn't you? ;)

image

**Comment by sfgeorge** [comment]
@gfaza Can you review this for me?

FYI the following 2 minor commits are new:
* [CS] Satiate the CI hound's newfound appetite for single over double-quotes a719c6a
* [CS] Appease the blood hound with a frozen string sacrifice 1d2ceb3

**Comment by gfaza** [comment]
👍

**Comment by sfgeorge** [comment]
Sorry, a few more commits to come down the pike very soon.

**Comment by gfaza** [comment]
np - i'll be watching

**Comment by gfaza** [comment]
👍

**Comment by sfgeorge** [comment]
@ajayanty Can you merge this for me?

**Pull request merged**